app/init/push_notifications.ts

```diff
@@ -52,25 +52,9 @@ class PushNotifications {
         Notifications.events().registerNotificationReceivedForeground(this.onNotificationReceivedForeground);
     }
 
-    cancelAllLocalNotifications = () => {
-        Notifications.cancelAllLocalNotifications();
-    };
-
-    cancelChannelNotifications = async (channelId: string, rootId?: string, isCRTEnabled?: boolean) => {
-        const notifications = await NativeNotifications.getDeliveredNotifications();
-        this.cancelNotificationsForChannel(notifications, channelId, rootId, isCRTEnabled);
-    };
-
-    cancelChannelsNotifications = async (channelIds: string[]) => {
-        const notifications = await NativeNotifications.getDeliveredNotifications();
-        for (const channelId of channelIds) {
-            this.cancelNotificationsForChannel(notifications, channelId);
-        }
-    };
-
     cancelNotificationsForChannel = (notifications: NotificationWithChannel[], channelId: string, rootId?: string, isCRTEnabled?: boolean) => {
```

Comment:
Shall we remove this function as it's not called anymore?

Reply:
I thought I removed it.. thanks